frontend/src/metabase/admin/tasks/containers/TasksApp.jsx

```diff
@@ -12,16 +13,22 @@ import Tooltip from "metabase/components/Tooltip";
 @Task.loadList({
   pageSize: 50,
 })
+@Database.loadList()
 class TasksApp extends React.Component {
   render() {
     const {
       tasks,
+      databases,
       page,
       pageSize,
       onNextPage,
       onPreviousPage,
       children,
     } = this.props;
+    const db_id_to_engine = {};
+    for (const db of databases) {
```

Comment:
For what it's doing, I don't think it's worth moving this to an outside function or putting it inside `connect` or anything, but I might just leave a little note about what's being done here and why so it stands out a bit more and it's easier to grok on subsequent viewing.